doc/btp_gap.txt

```diff
@@ -645,6 +645,189 @@ Commands and responses:
 
                 In case of an error, the error response will be returned.
 
+	Opcode 0x2a - Configure Advertising Set
+
+		Controller Index:	<controller id>
+		Command parameters:	Advertising Set ID (1 octet)
+					SID (1 octet)
+					Own Address Type (1 octet)
+					Advertising Type (1 octet)
+					Primary Advertising PHY (1 octet)
+					Secondary Advertising PHY (1 octet)
+					Secondary Advertising Maximum Skip (1 octet)
+					Coded PHY Options (1 octet)
+					Filter Accept List (1 octet)
+					Directed Advertising (1 octet)
+					Options (2 octets)
+					Minimum Interval, in 0.625ms units (4 octets)
+					Maximum Interval, in 0.625ms units (4 octets)
+					Peer Address Type (1 octet)
+					Peer Address (6 octets)
+		Return Parameters:	Configured Advertising Set ID (1 octet)
+
+		This command is used to create and configure advertising sets, which can
+		be used for both legacy and extended advertisements.
+
+		If the Advertising Set ID is 0xFF, a new advertising set will be created.
+
+		Otherwise, the command will configure the parameters of the specified
+		advertising set. The advertising set must be stopped when the command
+		is given, otherwise an error will be emitted.
+
+		The allowed values for Advertising Set ID are:
+			0x00...0xef = Use the specified advertising set
+			0xff = Create a new advertising set
+
+		The allowed values for SID are 0...15.
+
+		The allowed values for Own Address Type are:
+			0x00 = Identity Address
+			0x01 = Random Address
+
+		The allowed values for Advertising Type are a bit-wise OR of the
+		following bits:
+			0 = Connectable
+			1 = Scannable
+			2 = Extended Advertising
+			3 = Anonymous Advertising
+
+		The allowed values for Primary Advertising PHY are:
+			0x00 = 1M
+			0x01 = Coded
+
+			Ignored if Extended Advertising is not used.
+
+		The allowed values for Secondary Advertising PHY are:
+			0x00 = 1M
+			0x01 = 2M
+			0x02 = Coded
+
+			Ignored if Extended Advertising is not used.
+
+		The allowed values for Secondary Advertising Maximum Skip are:
+			0x00 = Do not skip any advertising events
+			0x01...0xff = Allow skipping up to the specified number of
+				      advertising events before sending AUX_ADV_IND
+
+			Ignored if Extended Advertising is not used.
+
+		The allowed values for Coded PHY Options are:
+			0x00 = No preferred coding type
+			0x01 = S=2 coding preferred
+			0x02 = S=8 coding preferred
+			0x03 = S=2 coding required
+			0x04 = S=8 coding required
+
+			Ignored if Coded PHY is not used.
+
+		The allowed values for Filter Accept List are a bit-wise OR of the
+		following bits:
+			0 = Use the filter accept list for scan requests
+			1 = Use the filter accept list for connections
+
+		The allowed values for Directed Advertising are:
```

Comment:
this could be HD selection flag only, and if directed to be used could be based on value of peer_addr